operator/pkg/util/yaml.go

```diff
@@ -72,6 +75,19 @@ func UnmarshalWithJSONPB(y string, out proto.Message, allowUnknownField bool) er
 		return err
 	}
 
+	// Remove creationTimestamp from jb if it exists
+	content := make(map[string]interface{})
+	err = json.Unmarshal(jb, &content)
```

Comment:
What is this for? Any reference to an issue? Can we add a test?

Reply:
The bug was https://github.com/istio/istio/issues/18615 .  The reason is that API Machinery defines CreationTimestamp with `json:"creationTimestamp,omitempty" protobuf:"bytes,8,opt,name=creationTimestamp"` and IstioOperator uses the other protobuf library.  Until we have a fix for this issue I don't know how to test it.

Reply:
That bug was supposedly fixed though. Where are you seeign this now?

Reply:
Using 1.6.0-beta.0, with Istio already installed, do `./bin/istioctl manifest apply -f <(kubectl -n istio-system get istiooperator installed-state -o yaml)` and you will see the problem.  (`generate` reports the same problem but it is less obvious because the error message is huge).

Reply:
Fixing in `UnmarshalWithJSONPB` won't work I think. This is a generic method. maybe it should be in UnmarshalIOP?

Reply:
Fixed.